.github/workflows/test.yaml

```diff
@@ -86,9 +89,9 @@ jobs:
           _numpy_version: ${{ matrix.numpy-version }}
         run: |
           python -um pip install --upgrade --upgrade-strategy eager wheel
-          python -um pip install --upgrade --upgrade-strategy eager coverage treelog stringly meshio numpy$_numpy_version
+          python -um pip install --upgrade --upgrade-strategy eager coverage numpy$_numpy_version
           # Install Nutils from `dist` dir created in job `build-python-package`.
-          python -um pip install --no-index --find-links ./dist nutils
+          python -um pip install ${{needs.build-python-package.outputs.wheel}}[import_gmsh]
```

Comment:
For consistency with the other `${{ ... }}` expressions, could you add spaces inside the braces?

Reply:
fixed